sample/src/sample/BasicConsumerProducerExample.java

```diff
@@ -152,7 +153,10 @@ private Runnable getConsumerTask() {
                 LOG.error(e.getMessage());
             } finally {
                 consumer.unsubscribe();
-                consumer.close();
+                try {
+                    consumer.close();
+                } catch (IOException e) {
+                }
```

Comment:
May be here we can log an exception or print the stacktrace. Like BasicPushConsumerExample.java  line 79.

Reply:
Done